<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_base3_ud</class>
 <widget class="QWidget" name="Form_base3_ud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_vueensemble">
     <property name="text">
      <string>Global view</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_cuve">
     <property name="text">
      <string>Trap view</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_poires">
     <property name="text">
      <string>Float view</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_vanne">
     <property name="text">
      <string>Valve view</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_armoire">
     <property name="text">
      <string>Controller view</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
